phpstan.neon

```diff
@@ -7,3 +7,8 @@ parameters:
     - uninstall.php
   scanFiles:
     - vendor/axepress/wp-graphql-stubs/wp-graphql-stubs.php
+  type_coverage:
+      return_type: 88
+      param_type: 77
```

Comment:
Can we use PHP Docs to define the param types and return types ..... since we are using PHPStan so these comments should help in coverage?

https://phpstan.org/writing-php-code/phpdocs-basics#properties

Reply:
The idea would be to move away from just documenting them (which, you're right, is also read by PHPStan), but actually let the PHP engine make use of them (and if necessary, complain about them) by moving out of the DocBlocks and into the parsed code as param and return types; PHPStan and other tools read them from there too, as do IDEs.

Reply:
@mehmoodak, is this actionable in this PR, and can you demonstrate or explain what you're proposing?
Thanks!